src/app/toolbar/toolbar.component.scss

```diff
@@ -308,4 +433,4 @@
     left: calc((1144px - 100vw) / 2);
     right: calc((1144px - 100vw) / 2);
   }
-}
+}
```

Comment:
```suggestion
}

```

Reply:
This is not resolved.

Reply:
When you see the red circle with a dash icon @kelvinkipruto, it means there is a missing new line at the end of a file. This is not resolved!